lib/import/helpers/brca/providers/rth/constants.rb

```diff
@@ -54,19 +54,19 @@ module Constants
             GENE_VALUES = [7, 8, 590, 18, 20, 865, 2744, 2804, 2808, 3186, 3394, 62, 3615, 3616, 76, 79, 358,
                            451, 577, 794, 1432, 1590, 1882, 2850, 3108, 3408, 5000, 5019, 72].freeze
 
-            RECORD_EXEMPTIONS = ['c.[-835C>T]+[=]', 'Deletion of whole PTEN gene',
-                                 'c.[-904_-883dup ]+[=]', 'whole gene deletion',
+            RECORD_EXEMPTIONS = ['c.( 442-127_ 593+118)', 'Deletion of whole PTEN gene', 'whole gene deletion',
                                  'Deletion partial exon 11 and exons 12-15', 'whole gene duplication'].freeze
 
             PROTEIN_REGEX = /p\.\[?\(?(?<impact>.+)(?:\))|
                              p\.\[(?<impact>[a-z0-9*]+)\]|
                              p\.(?<impact>[a-z]+[0-9]+[a-z]+)/ix
 
-            CDNA_REGEX = /c\.\[?(?<cdna>[0-9]+.+[a-z]+)\]?/i
+            CDNA_REGEX = /c\.\[?(?<cdna>[0-9-]+.+[a-z]+)\]?/i
```

Comment:
So I think you could also capture the +[=] on the variants if you wanted.

I'm not totally sure about the use of '+' (I thought it was supposed to be ';' but it may be a lab thing or old nomenclature style) but I think this is nomenclature to describe the variant as being heterozygous basically so it's saying the second allele (or strand of DNA) is normal. it's good information to capture either way.

Reply:
As discussed on Teams , we capture location string only and not symbols for cdna variant so won't be possible to capture this.